src/CircleGraph/CircleGraphCtrl.ts

```diff
@@ -153,12 +154,14 @@ export class CircleGraphCtrl {
           this.handleLoadModel(parseInt(message.offset));  // to number
           return;
         case MessageDefs.finishload:
-          this.handleFinishLoad();
+          this.handleFinishLoad(message);
```

Comment:
this change is needed to call `onViewMessage()` instead of `onFinishLoadModel()`